### ISSUE TITLE
vmcloak-install doesn't install any software --win7

### ISSUE THREAD

**Issue by lab313ru** [jbremer/vmcloak#55]
Trying to install `adobe9`: just *python.exe* empty window is visible in VirtualBox window.
There is no any Adobe processes created in VM, there is no something non-standard processes too.

**Comment by jbremer** [comment]
I have personally not really tested the 32-bit Windows 7 stuff yet.

**Comment by lab313ru** [comment]
Could you?

**Comment by razuz** [comment]
I'll take a run with that latest tomorrow ... lets see if I can reproduce that

**Comment by razuz** [comment]
just tested and it works fine on 32bit. 
@lab313ru Can you retry ? does this happen all the time you do "vmcloak install" and with any of packages ?

**Comment by razuz** [comment]
one thing you can try is to click "Ctrl+C" on the python window if that makes any difference

**Comment by lab313ru** [comment]
**Ctrl+C** just closes **python.exe** window.
And I cannot install any package with *install* option.

**Comment by razuz** [comment]
are you running on latest code from master ?

**Comment by lab313ru** [comment]
Latest at the day of creating this issue.
Host machine is Linux Mint.

**Comment by razuz** [comment]
sort of running out of ideas :) can vboxnet0 interface and vm see each other ? maybe some iptables rules are killing the comms ?

**Comment by lab313ru** [comment]
How to check it (ping is working)?

**Comment by razuz** [comment]
ping is one thing but you can also try to telnet to port 8000 against vm IP from mint side

**Comment by lab313ru** [comment]
But when installing Windows, it installs Python, so it works, doesn't it?

**Comment by razuz** [comment]
yes it should ... I'm quite impressed that it cannot install packages there and gets stuck

**Comment by lab313ru** [comment]
Could you make some log system for vmcloak, so we may find step, when it stucks?

**Comment by razuz** [comment]
we could do that ... but not this week

**Comment by wxfighting** [comment]
I got the same problem. I've created a win7 (x64) virtual machine with vmcloak (not fully automatically, I still need to manually click). When I ran "vmcloak install seven0 adobe9", I can see the virtual machine seven0 is running, but doing nothing.

**Comment by razuz** [comment]
are you running latest version ?

BR,
razu

**Comment by wxfighting** [comment]
Yes, I am running the latest version. The problem seems solved after I changed another window ISO file. It will be great if you can improve the logging. Anyway, vmcloack is and will be a great tool.

**Comment by razuz** [comment]
cool :) apart from couple other things I'm working to improve logging ... PR should be upcoming soon

**Comment by razuz** [comment]
it's just a wild guess, but I think this issue can be closed :) --debug flag has become quite verbose and points out possible issues nicely 
if there are any specific requests then I'd be happy to hear about them
